numpy/core/src/multiarray/lowlevel_strided_loops.c.src

```diff
@@ -1724,6 +1741,10 @@ mapiter_@name@(PyArrayMapIterObject *mit)
                         NPY_AUXDATA_FREE(transferdata);
                         return -1;
                     }
+                    if(subspace_ptrs[0] == self_ptr &&
+                       subspace_ptrs[1] == mit->extra_op_ptrs[0]) {
+                        skip <<= 1;
```

Comment:
skip = 1(fit the offset), 2 (twice, fit the stride), 4(the linear model is fitted, use predictions from this point on)